### ISSUE TITLE
can not stop app in ssl_hello_world example.

### ISSUE THREAD

**Issue by dcy** [ninenines/cowboy#1021]
Hi!
I can not stop the Erlang node with `./bin/ssl_hello_world stop` 
The log:

```
=ERROR REPORT==== 30-Aug-2016::17:35:04 ===
Error in process <0.185.0> on node 'ssl_hello_world_example@127.0.0.1' with exit value:
{{case_clause,{error,closed}},
 [{ranch_acceptor,loop,3,[{file,"src/ranch_acceptor.erl"},{line,28}]}]}
```

The error log shows ranch_acceptor here due to `{error, closed}` : https://github.com/ninenines/ranch/blob/master/src/ranch_acceptor.erl#L44

**Comment by essen** [comment]
This is a known issue with some Erlang versions. A quick workaround is to add the ssl dependency in ranch.app. A proper fix will be done in Ranch 2 (this is a breaking change).

**Comment by essen** [comment]
Should be fixed in both Ranch 1.3 and Cowboy 2.0.0-pre.4+. Thanks!
